### ISSUE TITLE
Add transpo-related sample data

### ISSUE THREAD

**Issue by matthewstern** [CMAP-REPOS/cmapplot#34]
@MMenninger is it possible for you to import a couple of sample datasets still? It would be great to have a few things in here that are transportation focused, so that we can make sure we test with other types of data besides for economic and demographic.

It took me longer than I want to admit to figure out how to load in and document sample data. I would be happy to sit down with you and walk you through what I did. In hindsight it wasn't complicated...just not obvious how to do it right.

**Comment by matthewstern** [comment]
@aidansd too!

**Comment by matthewstern** [comment]
@aidansd, thanks for uploading `redlight_camera.Rdata`. I commented on your commit [here](https://github.com/CMAP-REPOS/cmapplot/commit/f953214037b3f05e2463a9e77adc5ca9acc09ee3#commitcomment-38103750) but I wanted to recreate the issue here where it is more discoverable. 

Because of my weekend screw-up with PR #39, my recessions branch now includes your new dataset. So, when I ran `devtools::check()` on the branch before creating thew new PR #41, I noticed two warnings related to your dataset. It looks like, in addition to the documentation, one of your field names contains a weirdly-named string: 

```
-- R CMD check results -------------------------------------------- cmapplot 0.0.0.9000 ---- 
Duration: 54.3s  

> checking whether package 'cmapplot' can be installed ... WARNING   
    See below...  
> checking for missing documentation entries ... WARNING
    Undocumented code objects:
     'redlight'
   Undocumented data sets:
     'redlight'
   All user-level objects in a package should have documentation entries. 
   See chapter 'Writing R documentation files' in the 'Writing R   Extensions' manual.

> checking data for non-ASCII characters ... WARNING
     Warning: found non-ASCII string
     'o..INTERSECTION' in object 'redlight'

0 errors √  | 3 warnings x  | 0 notes √
```

> 
> 
> @aidansd Thanks for adding this! Rather than uploading anything directly to github, or changing any files in the master branch directly, we've been trying to follow the [feature-branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow). Although the notion of opening a feature branch just to add an example .rdata file may seem a bit silly, it's good form for when things get more complicated.
> 
> Additionally, @nmpeterson is being strict and requiring us to add documentation files for all our sample datasets. Check out, for example, the man file for the `economy_basic` dataset [here](https://github.com/CMAP-REPOS/cmapplot/blob/master/man/economy_basic.Rd). I learned the hard way that doing documentation is a total pain ([roxygen is a world upon itself](https://r-pkgs.org/man.html)) but you should be able have roxygen generate a new man file for this dataset by copying and pasting the documentation framework from one of the entries in [data.R](https://github.com/CMAP-REPOS/cmapplot/blob/master/R/data.R). (data.R does nothing except allow roxygen to generate man files for datasets when the package is compiled.)
> 
> Happy to touch base about any of this on Monday!!

**Comment by aidansd** [comment]
I figured as much-- the City of Chicago has some strange naming conventions that I thought wouldn't be an issue, but apparently are. I will clean a new version of the sample data and re-add to its own feature branch.

**Comment by matthewstern** [comment]
Oh, I noticed one other thing--the Rdata file is called `redlight_camera.RData` but the data file it loads is called is just called `redlight`. I think ideally they should be named identically?

**Comment by matthewstern** [comment]
@aidansd any chance you might be able to document your redlight data before you leave for Olympia? No worries if not!

**Comment by matthewstern** [comment]
I have deleted Aidan's branch because he did not document his added dataset before leaving CMAP. getting some transpo datasets in to the package remains a goal.

**Comment by matthewstern** [comment]
@dlcomeaux, PR #48 looked good, thanks for submitting it. I just merged into master. 

You'll notice another commit pre-merge... 

A useful QC tool Noel taught us earlier in this process was the function from the `devtools` package called `check()`, which runs a laundry list of really aggressive checks on a package being developed. We're trying to submit PRs only once the author has run the function and received 0 `errors` and no `warnings` or `notes` of significance. Then, whoever checks the PR for approval should confirm the same is true from a second machine. 

In this case, `devtools::check()` showed me a documentation error in a file that had nothing to do with your stuff... not sure how that snuck in, but I made a small tweak to fix.
